### DISCUSSION TITLE
请问下，为什么我的apisix在更换监听端口后，配置的Route需要再创建一遍才能生效

### DISCUSSION THREAD

**Discussion by woyouyitouchenfeijv** [apache/apisix#6359] [Q&A] [answered]
是不是我的配置不少了什么

**Comment by tokers** [comment]
Please provide your operation steps and the behaviors of APISIX after your operation.

**Reply by woyouyitouchenfeijv** [reply]
我用curl命令，创建一个routes
然后去修改了apisix的apisix.yaml 里加了端口监听，再重新启动
新的routes就不生效了，但是在dashboard里可以看到

**Reply by tokers** [reply]
Any suspected details in error.log?

**Comment by woyouyitouchenfeijv** [comment, marked as answer]
log err 500。It will take effect in 10 minutes。I think there are too many things in the machine and it's stuck

**Reply by tokers** [reply]
Could you paste some? It's difficult to troubeshoot only with current information.

**Reply by woyouyitouchenfeijv** [reply]
2022/02/21 19:04:57 [error] 15781#15781: *5330 lua entry thread aborted: runtime error: /usr/local/apisix/apisix/init.lua:362: attempt to index upvalue 'local_conf' (a nil value)
stack traceback:
coroutine 0:
        /usr/local/apisix/apisix/init.lua: in function 'http_access_phase'
        access_by_lua(nginx.conf:270):2: in main chunk, client: 1.1.1.1, server: _, request: "GET /xxxx?deptId=1 HTTP/1.1", host: "rean.name"


this is error。log
